packages/scheduler-sequelize/package.json

```diff
@@ -43,7 +43,8 @@
     "rrule-rust": "^1.2.0",
     "sequelize": "^6.6.2",
     "typed-emitter": "^1.3.1",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "lunartick-deprecated": "npm:@ordermentum/lunartick@0.0.19"
```

Comment:
dev dependency?

Reply:
I'm assuming this is interim while we transition

Reply:
Moved to dev dep. 
Yeah, can remove once the solution is prod tested.